### PR TITLE
Test revisited

### PR DESCRIPTION
**Pull request by namfal**
Removed some tests and reduced some loops. Also inscresed the timeout to 35s. Some tests had same setup (creating 3 subaccounts for instance) but differewnc API call/assertions. Instead of creating new set of data for each test, those tests now use a global data for a file.

I ran the tests 10 times, it failed once. I would say it is a big improvement!

### CODE REVIEW

**Comment by namfal** [comment]
Forgot to mention why we need these small tweaks: Tests on master on Travis fails frequently due to timeout. Last time I needed to merge a feature branch, I had to restart the tests several times.

**Comment by namfal** [comment]
@mroloux You can search for `beforeAll` and they should come up, but github is not showing all the diff. Mostly the tests about statusChanges, for instance `listStatusChangesAfter.test.js`

**Comment by namfal** [comment]
now that I know how to add pics

Screen Shot 2020-02-11 at 10 20 12

File: tests/events/listStatusChangesAfter.test.js [hunk 1]
```diff
@@ -1,240 +1,188 @@
...
-    await client.events.book(event.key, 'C-2')
-    let firstPage = await client.events.statusChanges(event.key).firstPage()
-
-    let params = new StatusChangesParams().withFilter('1')
-    let pageAfter = await client.events.statusChanges(event.key).pageAfter(firstPage.items[0].id, params)
-
-    expect(pageAfter.items).toEqual([])
+describe('list status changes after id', () => {
+    beforeAll(async () => {
+        jest.setTimeout(35000)
```
Comment (mroloux) [inline tests/events/listStatusChangesAfter.test.js]: Do we need to set a timeout here? Or should beforeAll timeout after the 30s we set in setupTests.js?

Comment (namfal) [reply tests/events/listStatusChangesAfter.test.js]: setupTest contains `beforeEach`, which runs after `beforeAll` so unless I am wrong, without the timeout, this beforeAll would use the default 500ms

**Comment by mroloux** [comment]
@namfal I merged in master. I removed the common setup in e.g. `listStatusChangesAfter.test.js`, since that would make the tests harder to read and maintain imho (for not much performance benefit).

Tests are green locally.

**Pull request merged**